src/components/[guild]/activity/ActivityLogAction/components/FormTag.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Button,
+  Circle,
+  forwardRef,
+  HStack,
+  Tag,
+  TagProps,
+  TagRightIcon,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import { DotsThreeVertical, Funnel, PencilSimpleLine } from "phosphor-react"
+import useSWRImmutable from "swr/immutable"
+import { ActivityLogType } from "../../ActivityLogContext"
+import {
+  FILTER_NAMES,
+  useActivityLogFilters,
+} from "../../ActivityLogFiltersBar/components/ActivityLogFiltersContext"
+import ClickableTagPopover from "./ClickableTagPopover"
+import ViewInFormResponses from "./ClickableTagPopover/components/ViewInFormResponses"
+
+type Props = FormTagProps & Omit<TagProps, "colorScheme">
+
+type FormTagProps = { formId: number; guildId: number }
+
+type ClickableFormTagProps = FormTagProps & {
+  activityLogType?: ActivityLogType
+  userId?: number
+}
+
+type FormField = {
+  id: string
+  question: string
+  isRequired: boolean
+  options?: string
+  type: string
+  allowOther: boolean
+}
+type FormType = {
+  name: string
+  description: string
+  fields: FormField[]
+  id: number
+  creatorUserId: number
+  guildId: number
+  createdAt: string
+  updatedAt: string
+}
+
+const FormTag = forwardRef<Props, "span">(
+  ({ formId, guildId, ...tagProps }: Props, ref): JSX.Element => {
+    const { data } = useSWRImmutable<FormType>(
+      `/v2/guilds/${guildId}/forms/${formId}`
+    )
+
+    const tagColorScheme = useColorModeValue("alpha", "blackalpha")
+    const imgBgColor = useColorModeValue("gray.700", "gray.600")
+
+    return (
+      <Tag
+        ref={ref}
+        colorScheme={tagColorScheme}
+        minW="max-content"
+        h="max-content"
+        {...tagProps}
+      >
+        {!data?.name ? (
+          "Unknown form"
+        ) : (
+          <HStack spacing={1}>
+            <Circle bgColor={imgBgColor} size={4}>
+              <PencilSimpleLine />
+            </Circle>
+            <Text as="span" w="max-content">
+              {data?.name}
+            </Text>
+          </HStack>
+        )}
+
+        <TagRightIcon as={DotsThreeVertical} />
```

Comment:
We should display the icon conditionally, just like we did inside `GuildTag` (we don't need to render it when the tag is in the filters bar)

![image](https://github.com/guildxyz/guild.xyz/assets/92519134/f4113ace-a4f1-4448-a562-38f5a3791637)